packages/gasket-plugin-nextjs/package.json

```
@@ -78,17 +78,17 @@
     "eslint-plugin-unicorn": "^55.0.0",
     "jest": "^29.7.0",
     "lodash.merge": "^4.6.2",
-    "next": "^14.2.24",
+    "next": "15.2.3",
```

Can we use the range here?
```suggestion
    "next": "^15.2.3",
```